.github/workflows/ci.yml

```
@@ -73,3 +73,26 @@ jobs:
       # However, we don't have many projects or targets in this workspace, so we
       # lower the number of agents to reduce spent compute time.
       number-of-agents: 4
+
+  sonarcloud:
+    needs: dte_coordinator
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

issue: Add Git fetch depth 0 as recommended by the SonarCloud Scan action
```suggestion
      - uses: actions/checkout@v3
        with:
          # necessary for SonarCloud Scan
          fetch-depth: 0
```